pkgs/development/tools/buf/default.nix

```diff
@@ -26,6 +26,8 @@ buildGoModule rec {
     ./skip_test_requiring_network.patch
     # Skip TestWorkspaceGit which requires .git and commits.
     ./skip_test_requiring_dotgit.patch
+    # Skips the invalid_upstream test as it is flakey.
```

Comment:
Can we add a note explaining that this is based on an upstream commit (maybe link to it) and that it thus can be removed when the package is updated to a version that includes that commit?

Reply:
Done